### ISSUE TITLE
frozenColumn option Is Missing

### ISSUE THREAD

**Issue by rameshsinger** [ghiscoding/Angular-Slickgrid#49]
I Need To Implement My First Column As Frozen When I search frozenColumn There Is No Option Like That ,frozenColumn Options In SlickGrid v2.2

**Comment by ghiscoding** [comment]
Angular-Slickgrid is a wrapper of the SlickGrid fork from 6pac. Frozen is not yet implemented in that fork, see this [issue 26](https://github.com/6pac/SlickGrid/issues/26)

I will close the issue, since this really has to be implemented in the Slickgrid core

**Comment by rameshsinger** [comment]
When I Use autoResize,enableAutoResize / gridContainerId Options With Multiple Grid With Tab In My App , First Grid Height Become 0px And Not Responsive Thanks In Advance

**Comment by ghiscoding** [comment]
I'm not sure to understand all of your comments but if you mean that you are creating multiple Bootstrap Tabs and grids in each tabs, it's actually a problem with SlickGrid itself. The DOM element of the grid container **must be visible** for it to be sized correctly on grid creation. In our project, we use route change to create the grid only after changing Tab. 

If you still wish to create a grid without the DOM element being visible, you can resize the grid with the `resizerService` after changing tab, that should work but I personally prefer to create the grid with a route change when changing tab.

If you have other totally different issue, please open separate issue.

**Comment by rameshsinger** [comment]
Thanks For Your Reply

**Comment by ghiscoding** [comment]
if you like and use the lib, you can also up vote ⭐️  
You don't have to do it, but if you do, it helps in making it more popular.

Thanks

**Comment by rameshsinger** [comment]
How To override an Editor Or Create My Custom Editor

**Comment by ghiscoding** [comment]
You should really read the [Wiki](https://github.com/ghiscoding/Angular-Slickgrid/wiki/Editors), I spent a lot of time writing them.

**Comment by rameshsinger** [comment]
Ok Thank You

**Comment by ghiscoding** [comment]
@rameshsinger 
Please note this has been implemented, see the [Frozen Example](https://ghiscoding.github.io/angular-slickgrid-bs4-demo/#/frozen)
